### ISSUE TITLE
Распознавание номеров машин с фото

### ISSUE THREAD

**Issue by freemanoid** [dziaineka/parkun-bot#1]
Есть идея добавить распознавание номеров машин с фото, будет ли интересен такой contribution?

**Comment by dziaineka** [comment]
> Есть идея добавить распознавание номеров машин с фото, будет ли интересен такой contribution?

Да, идея интересная. Я ее представлял себе как сервер с rest api. Бот посылает запрос с ссылкой на фото - получает номер и ненавязчиво предлагает пользователю его использовать.
